Dockerfile

```diff
@@ -11,9 +11,12 @@ RUN passwd -l root
 FROM base AS builder
 ARG version
 
+# The following line fix an issue with openjdk installation
+RUN mkdir -p /usr/share/man/man1
+
 # Get required apt packages
 RUN apt-get update \
-  && apt-get install -yy build-essential libffi-dev libfuzzy-dev \
+  && apt-get install -yy openjdk-8-jre-headless java-common build-essential libffi-dev libfuzzy-dev \
```

Comment:
As @cccs-sgaron pointed out this shouldn't be in the assemblyline-base image, would make more sense to place it in the assemblyline-v4-service image as the base for other services.

Reply:
I've added this to the service-base PR, just need to remove it from the assemblyline-base